### ISSUE TITLE
Received `false` for a non-boolean attribute `uncloseable`

### ISSUE THREAD

**Issue by DaleSalcedo** [georgeOsdDev/react-draggable-tab#101]
I'm trying to use this at the most basic level, but I'm getting the error:

> warning.js?6327:33 Warning: Received `false` for a non-boolean attribute `uncloseable`.
> 
> If you want to write it to the DOM, pass a string instead: uncloseable="false" or uncloseable={value.toString()}.
> 
> If you used to conditionally omit it with uncloseable={condition && value}, pass uncloseable={condition ? value : undefined} instead.

I've tried adding uncloseable={false} but that just gives me a different error.  and the example you guys have uses "unclosable" without the "e" so it gets even more confusing.

How can I fix this?

```
`class App extends React.Component {
  constructor(props) {
    super(props);
```

    this.state = {
      tabs:[
        (<Tab key={'tab0'} title={'Tab 1'}>
          <div>
            <h1>Tab Content 1</h1>
          </div>
        </Tab>),
        (<Tab key={'tab1'} title={'Tab 2'}>
          <div>
            <h1>Tab Content 2</h1>
          </div>
        </Tab>)
      ]
    };
  }

```
  render() {
    return (
      <Tabs
        tabs={this.state.tabs}
      />
    )
  }
}`
```

**Comment by ovaisahmed43** [comment]
+1

**Comment by georgeOsdDev** [comment]
Fixed with #102 and  react-draggable-tab@0.10.1 is published

> I've tried adding uncloseable={false} but that just gives me a different error. and the example you guys have uses "unclosable" without the "e" so it gets even more confusing.

Fixed typo `uncloseable` ->`unclosable`
